charts/kube-otel-stack/values.yaml

```diff
@@ -179,6 +179,15 @@ metricsCollector:
         check_interval: 1s
         limit_percentage: 75
         spike_limit_percentage: 30
+      metricstransform/k8sservicename:
```

Comment:
Is this the new way to do attribute processing?

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/metricstransformprocessor (open to other options as well)